src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Debugging/RazorBreakpointSpanEndpoint.cs

```diff
@@ -45,7 +45,7 @@ public Uri GetTextDocumentIdentifier(RazorBreakpointSpanParams request)
         }
 
         var codeDocument = await documentContext.GetCodeDocumentAsync(cancellationToken).ConfigureAwait(false);
-        var sourceText = await documentContext.GetSourceTextAsync(cancellationToken).ConfigureAwait(false);
```

Comment:
not for this PR, but should `DocumentContext.GetCodeDocumentAsync` populate `_sourceText` with this value as well? I assume we have the two separated for some optimization where you need sourcetext but not a full `RazorCodeDocument`

Reply:
Yes, that's a good idea.